test/unit/migrate_assets_to_asset_manager_test.rb

```diff
@@ -93,3 +96,29 @@ class OrganisationLogoFilePathsTest < ActiveSupport::TestCase
     end
   end
 end
+
+class OrganisationLogoFileTest < ActiveSupport::TestCase
```

Comment:
Minor: It feels a bit weird that this test of a nested class is in the top-level namespace. Is there any reason not to nest it inside the test case for the parent class?

Reply:
> Minor: It feels a bit weird that this test of a nested class is in the top-level namespace. Is there any reason not to nest it inside the test case for the parent class?

I copied the test that was already there for `OrganisationLogoFilePathsTest` in that it is also at the top-level rather than nested. I don't feel particularly strongly about it so I'm going to leave it as it is.